bin/capsule-certs-generate

```diff
@@ -45,11 +45,20 @@ CONFIG_CONTENT = <<EOF
 EOF
 
 config_file = temp_file('capsule-certs', CONFIG_CONTENT)
+
+# generate our parser cache and add it to the config file
+parser_cache = temp_file('capsule-certs.parser_cache', `kafo-export-params -c #{config_file.path} -f parsercache --no-parser-cache`)
```

Comment:
Can we do this on a production box? I assume it would require us to package puppet-strings which we were trying to avoid?